src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -112,51 +164,89 @@
             </v-col>
 
             <!-- Mailing Address -->
-            <v-col cols="12" :sm="isSummaryView ? 4 : 3" :class="{ 'removed': wasRemoved(orgPerson)}">
-              <MailingAddress class="peoples-roles-mailing-address" :address="orgPerson.mailingAddress" />
+            <v-col
+              cols="12"
+              :sm="isSummaryView ? 4 : 3"
+              :class="{ 'removed': wasRemoved(orgPerson)}"
+            >
+              <MailingAddress
+                class="peoples-roles-mailing-address"
+                :address="orgPerson.mailingAddress"
+              />
             </v-col>
 
             <!-- Delivery Address -->
-            <v-col cols="12" sm="3" :class="{ 'removed': wasRemoved(orgPerson)}" v-if="showDeliveryAddressColumn">
+            <v-col
+              v-if="showDeliveryAddressColumn"
+              cols="12"
+              sm="3"
+              :class="{ 'removed': wasRemoved(orgPerson)}"
+            >
               <template
                 v-if="hasRoleDirector(orgPerson) || hasRoleProprietor(orgPerson) || hasRolePartner(orgPerson)"
               >
                 <template v-if="IsSame(orgPerson.mailingAddress, orgPerson.deliveryAddress, ['id'])">
                   <span class="peoples-roles-delivery-address info-text">Same as Mailing Address</span>
                 </template>
                 <template v-else>
-                  <DeliveryAddress class="peoples-roles-delivery-address" :address="orgPerson.deliveryAddress" />
+                  <DeliveryAddress
+                    class="peoples-roles-delivery-address"
+                    :address="orgPerson.deliveryAddress"
+                  />
                 </template>
               </template>
             </v-col>
 
             <!-- Email Address -->
-            <v-col cols="12" sm="3" :class="{ 'removed': wasRemoved(orgPerson)}" v-if="showEmailColumn">
+            <v-col
+              v-if="showEmailColumn"
+              cols="12"
+              sm="3"
+              :class="{ 'removed': wasRemoved(orgPerson)}"
+            >
               <p>{{ orgPerson.officer.email }}</p>
             </v-col>
 
             <!-- Roles -->
-            <v-col cols="12" sm="2" :class="{ 'removed': wasRemoved(orgPerson)}" v-if="showRolesColumn">
+            <v-col
+              v-if="showRolesColumn"
+              cols="12"
+              sm="2"
+              :class="{ 'removed': wasRemoved(orgPerson)}"
+            >
               <!-- Warning if orgPerson has no roles -->
               <div v-if="orgPerson.roles.length > 0">
-                <v-col v-for="(role, index) in orgPerson.roles" :key="index" class="col-roles">
+                <v-col
+                  v-for="(role, index) in orgPerson.roles"
```

Comment:
This is also a build/lint warning. We can fix this later.

![image](https://user-images.githubusercontent.com/10002889/234071358-e2ad0173-eb69-4572-bc57-429113b81a64.png)